dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -97,6 +104,5 @@ else
         export WORKLOAD=$AIRFLOW_CTX_TASK_ID-must-gather
         ./ocp_diagnosis.sh > /dev/null
     fi
-    echo $UUID
```

Comment:
was this intentional ? I think we need this for [indexing](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/tasks/index/status.py#L51) job, unless if its printed by e2e or kube-burner scripts.

Reply:
woops, good catch, I wasn't aware of this trick, reverting this change